h2o-algos/src/main/java/hex/mojo/Generic.java

```diff
@@ -0,0 +1,98 @@
+package hex.mojo;
+
+import hex.ModelBuilder;
+import hex.ModelCategory;
+import hex.genmodel.*;
+import water.Key;
+import water.fvec.ByteVec;
+import water.fvec.Frame;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Generic model able to do scoring with any underlying model deserializable into a format known by the {@link GenericModel}.
+ * Only H2O Mojos are currently supported.
+ */
+public class Generic extends ModelBuilder<GenericModel, GenericModelParameters, GenericModelOutput> {
+
+    public Generic(GenericModelParameters genericParameters){
+        super(genericParameters);
+    }
+
+    public Generic(boolean startup_once) {
+        super(new GenericModelParameters(), startup_once);
+    }
+
+    @Override
+    public void init(boolean expensive) {
+        super.init(expensive);
+    }
+
+    @Override
+    protected Driver trainModelImpl() {
+        return new MojoDelegatingModelDriver();
+    }
+
+    @Override
+    public ModelCategory[] can_build() {
+        return ModelCategory.values();
+    }
+
+    @Override
+    public boolean haveMojo() {
+        return true;
+    }
+
+    @Override
+    public boolean isSupervised() {
+        return false;
```

Comment:
can we return the actual value from MOJO?

Reply:
We can't I guess. There is no specific MOJO known in this phase.